src/plugins/ini/README.md

```diff
@@ -3,7 +3,7 @@
 - infos/licence = BSD
 - infos/needs =
 - infos/provides = storage/ini
-- infos/recommends = binary
+- infos/recommends = base64
```

Comment:
In the clauses recommends/needs we should be as abstract as possible (so that mount also works if base64 is not present, but some other binary-transformation plugin is present). It is better to rewrite the tutorials.